crates/sargon/src/profile/logic/query_factor_sources.rs

```diff
@@ -61,54 +61,6 @@ impl Profile {
 
         explicit_main.unwrap_or(implicit_main).clone()
     }
-
-    fn next_derivation_index_for_entity_for_factor_source(
```

Comment:
Replaced by `NextDerivationEntityIndexAssigner`, used by `FactorInstancesProvider` and upgraded to handle FactorInstances with derivation paths where entity index is in `Securified` key space.